src/components/HomePage/SearchResultsArea/Offer/useChipsFieldSearch.spec.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import React from "react";
+import { MemoryRouter, useLocation } from "react-router-dom";
+import qs from "qs";
+
+import { renderWithStoreAndTheme, TestComponent } from "../../../../test-utils";
+import { act } from "react-dom/test-utils";
+import useChipsFieldSearch from "./useChipsFieldSearch";
+import { createTheme } from "@material-ui/core";
+
+describe("useChipsFieldSearch", () => {
+    const theme = createTheme({});
+
+    it("should change fields's search param when adding fields", async () => {
+        let addFieldWithUrl, location;
+        const initialState = {
+            offerSearch: {
+                searchValue: "searchValue",
+                jobDuration: [1, 2],
+                fields: [],
+                technologies: [],
+            },
+        };
+
+        const callback = () => {
+            addFieldWithUrl = useChipsFieldSearch().addFieldWithUrl;
+
+            location = useLocation();
+        };
+
+
+        renderWithStoreAndTheme(
+            <MemoryRouter initialEntries={["/"]}>
+                <TestComponent callback={callback} />
+            </MemoryRouter>,
+            { initialState, theme }
+        );
```

Comment:
I just now looked at that function, perhaps you could change to be more flexible (for instance, accepting a theme and an initial state). Also, it could/should be refactored away from Enzyme and into RTL, so we could kill 2 birds with 1 stone.

Reply:
:+1: